packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -160,8 +160,18 @@ var convertDate = function (dateString) {
   throw `Bad input for Date type in ${dateString}`;
 };
 
+var alreadyValidDateTime = function (dateTimeString) {
+  if (!dateTimeString || dateTimeString.toString() === "") return false;
+  // Match the format YYYY-MM-DD
```

Comment:
The regex is a "starts with", so I think its worth to write with this in mind.

Also, can we add it as a JSDoc? Not sure if VSCode is picking those up for .js files, but worth doing it right IMO just in case.

Reply:
added jsdoc
regex isnt a start with. Has the $ at the end